app/views/content/train-to-be-a-teacher/school-placements.md

```diff
@@ -20,8 +20,9 @@ quote:
     text: "My mentor was the special educational needs co-ordinator. This meant I was able to learn more about this role and improve my understanding of the strategies and support available for children with special educational needs and disabilities (SEND)."
     name: "Abbe, trainee teacher"
     classes: quote--indent
-promo_content:
-    - content/train-to-be-a-teacher/promos/adviser-promo-apply
+cta_adviser:
+  adviser:
+    text: An adviser with years of teaching experience can tell you more about school placements and how to submit a successful application. Chat by phone, text or email, as little or as often as you need.
```

Comment:
I don't think we can say `how to submit a successful application` as that depends on the candidate's eligibility etc. etc. (which is why we talk about `Maximise your chances of success.....` in other promos